app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -59,6 +60,52 @@ def in_person_enrollment?(user)
         # WILLFIX: After LG-6872 and we have enrollment saved, reference enrollment instead.
         ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
       end
+
+      def usps_proofer
+        if IdentityConfig.store.usps_mock_fallback
+          UspsInPersonProofing::Mock::Proofer.new
+        else
+          UspsInPersonProofing::Proofer.new
+        end
+      end
+
+      def create_usps_enrollment(enrollment)
+        pii = user_session[:idv][:pii]
+        applicant = UspsInPersonProofing::Applicant.new(
+          {
+            unique_id: enrollment.usps_unique_id,
+            first_name: pii.first_name,
+            last_name: pii.last_name,
+            address: pii.address1,
+            # do we need address2?
+            city: pii.city,
+            state: pii.state,
+            zip_code: pii.zipcode,
+            email: 'no-reply@login.gov',
+          },
+        )
+        proofer = usps_proofer
+
+        response = proofer.request_enroll(applicant)
+        response['enrollmentCode']
+      end
+
+      def save_in_person_enrollment(user, profile)
+        return unless in_person_enrollment?(user)
+
+        enrollment = InPersonEnrollment.create!(
+          profile: profile,
+          user: user,
+        )
+
+        enrollment_code = create_usps_enrollment(enrollment)
+        return unless enrollment_code
+
+        # update the enrollment to status pending
+        enrollment.enrollment_code = enrollment_code
+        enrollment.status = :pending
+        enrollment.save!
```

Comment:
Will need error handling here too, in case there's an existing pending enrollment.